source/js/components/join/join.jsx

```diff
@@ -54,7 +54,6 @@ export default class JoinUs extends React.Component {
 
     let signupState = classNames({
       'row': true,
-      'py-3': true,
```

Comment:
@alanmoo what are the 3 versions? You meant form, submission failed, and submission succeed?